backend/api/models.py

```diff
@@ -0,0 +1,48 @@
+import uuid
+from django.db import models
+from django.utils import timezone
+
+
+# Create your models here
+class Place(models.Model):
+    address = models.CharField(max_length=55, primary_key=True)
+    name = models.CharField(max_length=55)
+
+
+class Category(models.Model):
+    category_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=55)
+
+
+class Cashflow(models.Model):
+    cashflow_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    is_income = models.BooleanField()
+    overall_value = models.FloatField()
+    timestamp = models.DateTimeField(default=timezone.now)
+    categories = models.ManyToManyField(Category)
+    place = models.OneToOneField(Place, on_delete=models.DO_NOTHING, blank=True, null=True)
+
+
+class Item(models.Model):
+    item_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=55)
+    amount = models.FloatField()
+    cashflow = models.ForeignKey(Cashflow, on_delete=models.CASCADE, blank=True, null=True)
+    value = models.FloatField()
+
+
+class Account(models.Model):
+    account_id = models.CharField(max_length=55, primary_key=True)
+    cashflow = models.ForeignKey(Cashflow, on_delete=models.CASCADE)
```

Comment:
As indicated in the ER diagram, an account may have many cashflows, but one cashflow may only have one account. Hence, the foreign key should lie in the `Cashflow`, and not in the `Account`